src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -19,7 +18,7 @@
 use Tribe__Date_Utils as Dates;
 use Tribe__Utils__Array as Arr;
 
-abstract class By_Day_View extends View{
+abstract class By_Day_View extends View {
```

Comment:
:no_entry_sign: Missing doc comment for class By_Day_View

<sub>hash: 644dfdbca53f2b3b41dae795182a3a98</sub>